examples/counter-no-graphql/src/service.rs

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) Zefchain Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+#![cfg_attr(target_arch = "wasm32", no_main)]
+
+mod state;
+
+use linera_sdk::{linera_base_types::WithServiceAbi, views::View, Service, ServiceRuntime};
+
+use self::state::CounterState;
+
+pub struct CounterService {
+    state: CounterState,
+}
+
+linera_sdk::service!(CounterService);
+
+impl WithServiceAbi for CounterService {
+    type Abi = counter_no_graphql::CounterNoGraphQlAbi;
+}
+
+impl Service for CounterService {
+    type Parameters = ();
+
+    async fn new(runtime: ServiceRuntime<Self>) -> Self {
+        let state = CounterState::load(runtime.root_view_storage_context())
+            .await
+            .expect("Failed to load state");
+        CounterService { state }
+    }
+
+    async fn handle_query(&self, _request: ()) -> u64 {
```

Comment:
We need a demo app that actually processes queries: either to answer queries or to support block building (aka mutations in GraphQL speak).

Reply:
Ok, I extended the request type which is now an enum.
The first case just returns a value, the second test creates an operation with `schedule_operation`.